config/routes.rb

```diff
@@ -13,7 +13,9 @@
     resources :communities, only: [:index, :show]
     resources :contact_forms, only: [:new, :create], path_names: { new: 'new_message' }
     resources :job_offers, only: [:index, :show]
-    resources :people, only: [:index, :show]
+    resources :people, only: [:index, :show] do
+      get '/:link_name', to: 'external_link#show', as: 'external_link'
```

Comment:
You can add constraints to `link_name` to be much secure here.

For example:
```ruby
constraints subject_type: /booking|external_booking/ do
      resources :charges,
                only: %i[new create index],
                path: ':subject_type/:subject_id'
    end
```

@torrocus WDYT?

Reply:
We could also define this names in one place, in some class, and just use them here.